package.json

```diff
@@ -366,18 +276,6 @@
 				"key": "alt+v",
 				"mac": "alt+v",
 				"when": "editorTextFocus && editorLangId == lean"
-			},
-			{
-				"command": "lean.infoView.displayGoal",
-				"key": "ctrl+shift+enter",
```

Comment:
This command (and in particular the keycombo) should continue to work.

Reply:
Yes it does still work using `lean.displayGoal` instead of `lean.infoview.diaplayGoal`